location/src/androidLibMain/kotlin/location/AndroidLocationFlowable.kt

```diff
@@ -92,17 +92,20 @@ sealed class LocationFlowableState(override val repo: LocationManagerStateRepo)
     }
 }
 
-actual class LocationFlowable :
-    BaseLocationFlowable() {
+actual class LocationFlowable : BaseLocationFlowable() {
+
+    class Builder(private val provider: FusedLocationProviderClient) : BaseLocationFlowable.Builder {
+        override fun create() = LocationFlowable().setFusedLocationProviderClient(provider)
+    }
 
     private var stateRepo = LocationManagerStateRepo(this)
 
-    fun setFusedLocationProviderClient(provider: FusedLocationProviderClient) {
+    private fun setFusedLocationProviderClient(provider: FusedLocationProviderClient) = apply {
```

Comment:
I suspect the state stuff in LocationFlowable is now mostly obsolete with the builder pattern (since it's guaranteed to be present after build).